desk/app/activity.hoon

```diff
@@ -190,7 +190,7 @@
       ~(tap by threads.unread)
     |=  [parent=message-key:ch [key=message-key:ch count=@ud]]
     ^-  (unit (list [time incoming-event:a]))
-    =/  writ=(unit writ:ch)  (~(get by wit.pact) time.parent)
+    =/  writ=(unit writ:ch)  (get:on:writs:ch wit.pact time.parent)
```

Comment:
These are classic, and nasty. Using `+by` on a `mop` will compile and get you results, but potentially the wrong result. Good catch.